Makefile.in

```diff
@@ -46,7 +46,7 @@ default: display-build-env add-git-hooks mana_prereqs
 all: default
 
 mana: dmtcp
```

Comment:
I am not sure why we are doing a make install here. Shouldn't that go under the `install` target?

Also, should we just remove this target altogether.

Reply:
@karya0 , I'm not sure I understand your point here.  We want to capture the default target for 'make'.  The conventional name for that would be `default` and not `install`.  Under that target, it's possible to call `make install` if desired.  Something like that seems to be happening here.